Dockerfile

```diff
@@ -42,6 +41,7 @@ COPY --chown=default:root --from=builder /usr/src/app/backend/package.json /usr/
 COPY --chown=default:root --from=builder /usr/src/app/backend/package-lock.json /usr/src/app/backend/package-lock.json
 COPY --chown=default:root --from=builder /usr/src/app/backend/node_modules /usr/src/app/backend/node_modules
 COPY --chown=default:root --from=builder /usr/src/app/backend/dist /usr/src/app/backend/dist
+COPY --chown=default:root --from=builder /usr/src/app/node_modules /usr/src/app/node_modules
```

Comment:
Why do we need the main node_modules? production ready bundle doesn't need it right? or now that we've mode some dependencies this has changed?

Reply:
It's needed right now because backend doesn't create a bundle. It compiles the typescript to javascript and then depends on node for resolution. We should move to bundling but that will need to be done later.
This will increase the size of what is included because it now has more than just the modules that are required by the backend.

The reason for the inclusion is because the monorepo hoists node_modules to the root for common / non-conflicting modules.